man/systemd-nspawn.xml

```diff
@@ -1060,6 +1078,42 @@
         same as the one reported on the host.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--console=</option><replaceable>MODE</replaceable></term>
+
+        <listitem><para>Configures how to set up standard input, output and error output for the container payload, as
+        well as the <filename>/dev/console</filename> device for the container. Takes one of
+        <option>interactive</option>, <option>read-only</option>, <option>passive</option> or <option>pipe</option>. If
+        <option>interactive</option> a pseudo-TTY is allocated and made available as <filename>/dev/console</filename>
+        in the container. It is then continously bi-directionally connected to the standard input and output passed by
+        the calling process.  <option>read-only</option> is similar but only the output of the container is propagated
```

Comment:
in this case, the calling process is always systemd-nspawn, right ? if yes, it might be clearer to say that explicitely...

Reply:
no, it's the caller of nspawn itself that is meant here. The man page is generally written from perspective of nspawn. I'll slightly reword this to make this clearer.